app/components/app_patient_session_search_result_card_component.rb

```diff
@@ -25,8 +25,8 @@ class AppPatientSessionSearchResultCardComponent < ViewComponent::Base
 
             if status_tag
               summary_list.with_row do |row|
```

Comment:
I think it might be possible to pass the row structure straight in to `with_row`, something like `with_row(status_row)`, maybe?

https://govuk-components.netlify.app/components/summary-list/#building-a-summary-list-directly-from-data

Reply:
Doesn't appear to work - `with_row` only takes a block.

Reply:
Hmm I wonder if it only works if all the rows are provided in that hash format?

Reply:
Looks like the only way is where one does `govuk_summary_list(rows: rows)` and then `rows` looks something like:

```
{
  rows: [
    { key: { text: "Name" }, value: { text: "Hercule Poirot" } },
    {
      key: { text: "Address" },
      value: { text: "Flat 203, 56B Whitehaven Mansions, Charterhouse Square, London" },
      actions: [{ href: "#", visually_hidden_text: "address" }]
    }
  ]
}
```

but our component is a bit more bespoke than that I think.